docs/reference/commandline/ps.md

```diff
@@ -51,7 +51,47 @@ The currently supported filters are:
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
```

Comment:
I did not add a line here for `ancestor` filter because the line is in #14570 :wink: